Make TTL configurable (#724)
neo4j/graph-data-science-client: doc/modules/ROOT/pages/gds-session.adoc

- **DBMS connection**.
This is a `DbmsConnectionInfo` object that contains the URI of an Neo4j instance, a username, and a password.

- **TTL**.

> This optional parameter specifies the time-to-live of the session. The session will be automatically deleted after the TTL has expired after the last user activity.

Reviewer: ```suggestion
This optional parameter specifies the time-to-live of the session. The session will be automatically deleted if the session was unused for the provided duration.
```

    session_name=name,
    memory=memory,
    db_connection=db_connection_info,

>     ttl=timedelta(hours=5),

Reviewer: how about we also add it to the notebook?

>     ttl=timedelta(hours=5),

Author: We are currently not describing the example at all. I think that if we add this line, we would also have to describe the rest of the parameters, which would be a duplication of the description above.
